backend/src/services/userService.ts

```diff
@@ -90,6 +100,30 @@ const checkPassword = async (
     }
 };
 
+const changePassword = async (
+    userID: string,
```

Comment:
Instead of userID take the userName as an argument here. Using the name is ok here since that must be unique as well even though it may change over time.